test/core/GnosisSafe.Estimation.spec.ts

```diff
@@ -17,24 +17,23 @@ describe("GnosisSafe", async () => {
                 }
             }`
         const reverter = await deployContract(user1, reverterSource);
+        const decoderSource = `
+            contract Decoder {
+                function decode(address to, bytes memory data) public returns (bytes memory) {
+                    (bool success, bytes memory data) = to.call(data);
+                    require(!success, "Shit happens");
```

Comment:
Maybe the message could be _better_

Reply:
It is just for testin :P ... a little bit of fun :D